packages/select/test/select.test.js

```diff
@@ -402,29 +384,6 @@ describe('vaadin-select', () => {
         expect(overlay.opened).to.be.false;
       });
 
-      it('should restore focused state on closing the overlay if phone', async () => {
-        select._phone = true;
-        await nextUpdate(select);
-        click(select._items[1]);
-        expect(select.hasAttribute('focused')).to.be.true;
-      });
-
-      it('should focus the button on closing the overlay if phone', async () => {
-        const focusedSpy = sinon.spy(valueButton, 'focus');
-        select._phone = true;
-        await nextUpdate(select);
-        click(select._items[1]);
-        await nextUpdate(select);
-        expect(focusedSpy.called).to.be.true;
-      });
-
-      it('should focus the button before moving the focus to next selectable element', async () => {
```

Comment:
Replaced this with better tests using `sendKeys` that actually ensure that focus is eventually moved to the first / last global focusable on overlay item <kbd>Tab</kbd> and <kbd>Shift</kbd> + <kbd>Tab</kbd> respectively.